sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -80,6 +81,25 @@ pi_result cuda_piPluginGetLastError(char **message) {
   return ErrorMessageCode;
 }
 
+// Returns plugin specific backend option.
+// Current support is only for optimization options.
+// Return empty string for cuda.
+// TODO: Determine correct string to be passed.
+pi_result cuda_piPluginGetBackendOption(pi_platform platform,
```

Comment:
```suggestion
pi_result cuda_piPluginGetBackendOption(pi_platform,
```
Unused arguments will turn into build errors

Reply:
Hi @smaslov-intel 

Thanks a lot for feedback. i have removed the unused arguments. There was a type (extra parentheses) that caused build fail and I resolved it. Can you please approve if everything looks ok?

Thanks

Reply:
approved